Code/Editor/Include/IFileUtil.h

```diff
@@ -80,6 +80,7 @@ struct IFileUtil
         FILE_TYPE_SCRIPT,
         FILE_TYPE_SHADER,
         FILE_TYPE_BSPACE,   // added back in 3.8 integration, may not end up needing this.
+        FILE_TYPE_SYSTEM,
```

Comment:
nit - from the code it's clear that this is going to use the OS file type associations to open the file but having a comment here to explain it would be nice

Reply:
Got a simple way as @cgalvan said.